src/site/layouts/office.drupal.liquid

```diff
@@ -0,0 +1,102 @@
+{% assign entryname = "public-outreach-materials" %}
```

Comment:
I think we can do `{% assign private = true %}` to exclude this from the sitemap for now.